.travis.yml

```diff
@@ -6,3 +6,4 @@ rvm:
   - 2.5.0
 script:
   - bundle exec rspec
+  - bundle exec rspec spec/lib/instrumentation/as_notifications.rb
```

Comment:
If breaking out the spec is the only way to do it, chaining them with `&&` might be preferable (https://docs.travis-ci.com/user/customizing-the-build/#Customizing-the-Build-Step) as it would always generate testing output from both sets but still fail properly. 

That said, I think your test tear-down could alias the method back to the original and remove the newly added method easily enough and eliminate the need for the second running of rspec.